conf/routes

```diff
@@ -62,6 +62,7 @@ PUT        /userdata/editionsClipboard               controllers.UserDataControl
 PUT        /userdata/frontIds                        controllers.UserDataController.putFrontIds()
 PUT        /userdata/frontIdsByPriority              controllers.UserDataController.putFrontIdsByPriority()
 PUT        /userdata/favouriteFrontIdsByPriority     controllers.UserDataController.putFavouriteFrontIdsByPriority()
+PUT        /userdata/featureSwitch                   controllers.UserDataController.putFeatureSwitch()
```

Comment:
do we also want a GET endpoint?

Reply:
As discussed, the current implementation of the client doesn't make use of any GET endpoints to retrieve user data.